src/pages/publication/EditPublication.tsx

```diff
@@ -60,7 +60,7 @@ const EditPublication: FC = () => {
 
   return (
     <>
-      <PageHeader>{t('publication:new_publication')}</PageHeader>
+      <PageHeader>{t(identifier ? 'edit_publication' : 'new_publication')}</PageHeader>
```

Comment:
<img width="1555" alt="Skjermbilde 2020-09-15 kl  10 40 58" src="https://user-images.githubusercontent.com/6595982/93187372-0742b000-f740-11ea-9927-b16857af69dc.png">

Skal det være så stor skrift? Synes det var litt i overkant eller?

Reply:
jo, kanskje... samme med linja under - syns egentlig `3px` ble _litt_ for mye, men `2px` ble litt lite igjen 😅 
du kan godt endre litt i conosle og se om du finner bedre komboer - jeg er åpen for det meste
Kine burde egentlig sett på dette også

Reply:
ser hva du mener ja! jeg synes 2.1rem var bra størrelse, men du liker kanskje ikke desimaltall? :P

Reply:
JEG HATER  DESIMALTALL!!!111!!1
neida, kan godt dra ned litt ja

Reply:
😂 😂 😂